impeller/renderer/backend/gles/proc_table_gles.h

```diff
@@ -99,6 +101,14 @@ struct GLProc {
   ///
   bool log_calls = false;
 
+  //----------------------------------------------------------------------------
+  /// Whether the OpenGL call asserts it is only used from / one thread in
+  /// IMPELLER_DEBUG builds.
```

Comment:
IMPELLER_DEBUG is debug/profile mode.  !defined(NDEBUG) means unopt local engine IIRC, that should be fine.